js/private/test/node-patches/BUILD.bazel

```diff
@@ -1,13 +1,19 @@
 load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")
+load("@npm//:@babel/cli/package_json.bzl", babel_bin = "bin")
 load("//js:defs.bzl", "js_test")
 
 TESTS = [
```

Comment:
maybe a bit cleaner to define MJS_TESTS and put `escape.js` in its own list so you don't need to filter below? also as written you will run escape.js in CJS_TESTS as well, is that intended?

Reply:
Yeah that reduces the code a bit, updated 👍